### ISSUE TITLE
Validation shouldn't randomly crop the input image

### ISSUE THREAD

**Issue by nitinsurya** [kentsyx/Neural-IMage-Assessment#15]
I haven't read the paper yet, but looking into the code, [random cropping](https://github.com/kentsyx/Neural-IMage-Assessment/blob/master/main.py#L38) on the validation set doesn't seem like a proper evaluation method, as this is usually done to avoid overfitting on training data.

**Comment by nitinsurya** [comment]
Came across this [RandomResizedCrop](https://pytorch.org/docs/stable/torchvision/transforms.html#torchvision.transforms.RandomResizedCrop), which would probably yield even better results.

**Comment by huanmofazong** [comment]
Yes, Ramdomcrop should be removed here when in test and val , otherwise randomness will be generated when test or val
